awesome_dashboard/static/src/dashboard.js

```diff
@@ -2,9 +2,12 @@
 
 import { Component } from "@odoo/owl";
 import { registry } from "@web/core/registry";
+import { Dashboard } from "./dashboard/dash_board";
 
 class AwesomeDashboard extends Component {
     static template = "awesome_dashboard.AwesomeDashboard";
+
+    static components = { Dashboard }
```

Comment:
```suggestion
    static components = { Dashboard };
```
Use `;` for variables which are not a function